evergreen/evergreen.yml

```diff
@@ -218,7 +218,9 @@ functions:
           export FLE_AWS_ACCESS_KEY_ID=${FLE_AWS_ACCESS_KEY_ID}
           export FLE_AWS_SECRET_ACCESS_KEY=${FLE_AWS_SECRET_ACCESS_KEY}
           ${PREPARE_SHELL}
+          evergreen/add-certs.sh
```

Comment:
The ticket says:
>we should convert our existing TLS tests so that we add the relevant certificates to the Windows system store

but if I'm not mistaken, we will add this certificate for all tests (not only the tests with configured AUTH). Is it ok?
Sorry, If I missed something

Reply:
Nope, you're correct! I think it does no harm however to add the certs even when they're not needed because we're removing them after the test run, and even if the test run crashes, it's highly unlikely that the added cert(s) will interfere with another project's tests. I think it's simpler to always add and remove the certs, but if you feel strongly about it, I can make the change :)

Reply:
To be honest, I don't know how it can affect us, but most likely we won't suffer from it. So, I'm ok with the current implementation